src/App.tsx

```diff
@@ -1,61 +1,126 @@
 import './App.scss';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo, TodoWithUser } from './react-app-env';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const getUserById = (id: number) => {
+  return usersFromServer.find(user => user.id === id) || null;
+};
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+  const [isTitleEmpty, setIsTitleEmpty] = useState(false);
```

Comment:
to me it seems that this case does not deserve to be a separate state, we can do it just by title.length

Reply:
if we use title.length the error message will be shown immediately after launch. But we need to see this message after button click

Reply:
kk, agree with you